src/Resources/views/payWithPaypal.html.twig

```diff
@@ -268,7 +268,7 @@
                     method: 'post',
                     headers: {},
                     body: error
-                }).then(window.location.reload());
+                }).then(() => window.location.reload());
```

Comment:
```suggestion
                }).then(window.location.reload);
```
`window.location.reload` is a function
`window.location.reload()` is the function call